src/components.js

```diff
@@ -155,7 +156,7 @@ const code = props => {
         </Notes>
       )
     default:
-      return <Pre {...props} />
+      return <Code {...props} />
```

Comment:
Would be good to keep the basic pre styles/theming intact for the new component – maybe it renders a default `<pre>` tag with no syntax highlighting if there's no `highlight` field in the theme?

Reply:
Makes sense.

Reply:
Just checked, we only know when inside the `Code` component. The problem is that if there is no `highlight` field we fall back to the default theme which is defined at 
https://github.com/busypeoples/mdx-deck/blob/fe63e5b7d67c47fdc5522409b51e279a3636173a/src/Code.js#L5

I choose a random default theme, but we could use the prism default f.e.

Reply:
Yeah, if this `code` component was wrapped with `withTheme` then it should have access to the theme, right?

Reply:
On the other hand this means themes have to be defined explicitly inside an mdx, would we even need a default theme inside `Code` then?